addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -877,6 +875,10 @@
             // Prevent icon buttons to shrink
             &.o_we_icon_button, &.fa {
                 flex: 0 0 auto;
+
+                &:not(.o_we_img_animate) {
+                    text-align: center;
```

Comment:
Button text should be centered...
![image](https://user-images.githubusercontent.com/8558351/234521267-0c8f34c9-db6a-4519-a62b-3d07cd8e1048.png)

![image](https://user-images.githubusercontent.com/8558351/234522120-cfb2da99-b457-4a71-a757-297b2816d76f.png)